docs/querying/operators.md

```diff
@@ -271,3 +271,6 @@ highest to lowest.
 Operators on the same precedence level are left-associative. For example,
 `2 * 3 % 2` is equivalent to `(2 * 3) % 2`. However `^` is right associative,
 so `2 ^ 3 ^ 2` is equivalent to `2 ^ (3 ^ 2)`.
+
+Only the operators who do not alter the values (level 4, 5 and 6 in the list
```

Comment:
We cover this topic above already, so it should get a mention in relation to arithmetic operators up in between an instant vector and a scalar above - rather than adding a new place.

This also has nothing to do with precedence.